src/Workspaces/Core/Portable/Execution/Checksum_Factory.cs

```diff
@@ -24,6 +24,18 @@ public static Checksum Create(Stream stream)
             }
         }
 
+        public static Checksum Create(string kind, Checksum checksum)
+        {
+            using (var stream = SerializableBytes.CreateWritableStream())
```

Comment:
Let's just leave it simple for now. We can do that if we need to optimize more and nothing else left.